index.bs

```diff
@@ -297,7 +297,7 @@ convenience of implementors two separate CDDL definitions are defined; the
 <dfn>remote end definition</dfn> which defines the format of messages produced
 on the [=local end=] and consumed on the [=remote end=], and the <dfn>local end
 definition</dfn> which defines the format of messages produced on the [=remote
-end=] and consumed on the [=local end=]
+end=] and consumed on the [=local end=].
```

Comment:
While you modify this line lets try to keep `[=remote end=]` in a single line.

Reply:
Done